docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-breathe<4.13.0
+breathe>4.14.0
```

Comment:
we should probably bound sphinx here as well

Reply:
Done, thanks!